core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -91,13 +91,19 @@ object ReassignPartitionsCommand extends Logging {
     if (duplicateReassignments.nonEmpty)
       throw new AdminCommandFailedException("Broker list contains duplicate entries: %s".format(duplicateReassignments.mkString(",")))
     val topicsToMoveJsonString = Utils.readFileAsString(topicsToMoveJsonFile)
+    val assignments = generateAssignment(zkUtils, brokerListToReassign, topicsToMoveJsonString)
```

Comment:
You can do `val (proposedAssignments, currentAssignments) = ...` and avoid the ugly `_2` and `_1` in subsequent lines.